hardware_isolation_entry.cpp

```diff
@@ -32,5 +38,33 @@ Entry::Entry(sdbusplus::bus::bus& bus, const std::string& objPath,
     this->emit_object_added();
 }
 
+namespace utils
+{
+
+EntrySeverity getEntrySeverityType(const openpower_guard::GardType gardType)
+{
+    switch (gardType)
+    {
+        case openpower_guard::GardType::GARD_Unrecoverable:
+        case openpower_guard::GardType::GARD_Fatal:
+            return EntrySeverity::Critical;
+
+        case openpower_guard::GardType::GARD_User_Manual:
+            return EntrySeverity::Manual;
+
+        case openpower_guard::GardType::GARD_Predictive:
+            return EntrySeverity::Warning;
+
+        default:
+            log<level::ERR>(
+                fmt::format("Unsupported GardType [{}] was given ",
+                            "to get the hardware isolation entry severity type",
+                            gardType)
+                    .c_str());
+            return EntrySeverity::Warning;
```

Comment:
I think an error should be raised

Reply:
There is unused GardType right? The host application can use any of those should we throw the exception or Continue?
Unused in BMC
```
GARD_Power = 0xE9,
GARD_PHYP = 0xEA,
GARD_Reconfig = 0xEB,
GARD_Void = 0xFF,
```